ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/machine/MachineManager.java

```diff
@@ -21,6 +23,26 @@
 public interface MachineManager {
```

Comment:
@vparfonov We still have "Destroy machine" and "Restart machine" actions, so I can not to remove this class